bigframes/session/__init__.py

```diff
@@ -342,6 +342,35 @@ def read_gbq_query(
         ``row_number() over ()`` if there is no natural unique index or you
         want to preserve ordering.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            Preserve ordering in a query input.
+
+            >>> df = bpd.read_gbq_query('''
```

Comment:
This example looks a bit advanced, would be nice to add a simpler one as the first example while keeping this as the second example, e.g. 

read_gbq_query("SELECT pitcherFirstName, pitcherLastName, pitchSpeed FROM `bigquery-public-data.baseball.games_wide`")